src/internal/modules/billing/forms/billing-batch-list.js

```diff
@@ -1,13 +1,7 @@
 const { formFactory, fields } = require('shared/lib/forms/');
+const Joi = require('@hapi/joi');
```

Comment:
See comment below, not sure this form is needed

Reply:
form removed